contributed_definitions/nyaml/NXopt.yaml

```diff
@@ -56,348 +43,675 @@ NXopt(NXobject):
+          #    Unit category: NX_ANGLE
+          #  - **angular1**: Slow-axis angular coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
+          #    Unit category: NX_ANGLE
+          #  - **spatial0**: Fast-axis spatial coordinate (or second slow axis if spatially integrated)
+          #    Unit category: NX_LENGTH
+          #  - **spatial1**: Slow-axis spatial coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
+          #    Unit category: NX_LENGTH
+          #  - **delay**: Calibrated delay time. Unit category: NX_TIME (s).
+          #  - **polarization_angle**: Linear polarization angle of the incoming or
+          #    outgoing beam.
+          #    Unit category: NX_ANGLE (° or rad)
+          #  - **ellipticity**: Ellipticity of the incoming or outgoing beam.
+          #    Unit category: NX_ANGLE (° or rad)
+          #  - **time_of_flight**: Total time of flight. Unit category: NX_TIME_OF_FLIGHT
+          #  - **time_of_flight_adc**: Time-of-flight values, analog-to-digital converted.
+          #  - **external_AXIS**: Describes an axis which is coming from outside the detectors scope.
+          \@signal:
+            enumeration: [raw]
+          raw(NX_NUMBER):
             doc: |
-              Specify the position of the window in the beam path by pointing
-              to the preceding element in the sequence of beam path elements.
-          window_effects_corrected(NX_BOOLEAN):
-            doc: |
-              Was a window correction performed? If 'True' describe the window
-              correction procedure in 'window_correction/procedure'.
-          window_correction(NXprocess):
-            exists: optional
-            doc: |
-              Was a window correction performed? If 'True' describe the
-              window correction procedure in ''
-            procedure:
-              doc: |
-                Describe when (before or after the main measurement + time
-                stamp in 'date') and how the window effects have been
-                corrected, i.e. either mathematically or by performing a
-                reference measurement. In the latter case, provide the link to
-                to the reference data in 'reference_data_link'.
-            reference_data_link:
-              exists: optional
-              doc: |
-                Link to the NeXus file which describes the reference data if a
-                reference measurement for window correction was performed.
-                Ideally, the reference measurement was performed on a reference
-                sample and on the same sample, and using the same conditions as
-                for the actual measurement with and without windows. It should
-                have been conducted as close in time to the actual measurement
-                as possible.
-          material(NX_CHAR):
+              Raw data before calibration.
+      (NXmonochromator):
```

Comment:
`NXmonochromator` is already optionally part of `NXinstrument`, no need to restate here.

Reply:
just a few specifications are right now in NXmonochromator